Ombi.Api/TheMovieDbApi.cs

```diff
@@ -100,5 +105,33 @@ public TmdbMovieDetails GetMovieInformationWithVideos(int tmdbId)
             var movies = await Client.GetMovie(imdbId);
             return movies ?? new Movie();
         }
+
+        public async Task<List<Movie>> SearchActor(string searchTerm)
+        {
+            SearchContainer<SearchPerson> result = await Client.SearchPerson(searchTerm);
+            var person = result?.Results[0] ?? null;
```

Comment:
Sometimes, more than one person could be returned.  They **seem** to return the most popular actor first, so this just grabs that person.  I was looping through the actors at first and hitting each of them, but (obviously) that causes even more hits to the API.  This was basically an easy hack as part of a first pass.